src/types/models/chats.ts

```diff
@@ -0,0 +1,10 @@
+import { db }  from '../../utils/db'
+
+interface Chats {

```

Comment:
I'm not sure where the lint rule went but this should be a `type` instead of an interface

Reply:
sure. The money model also has an `interface` iirc